src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -76,6 +76,18 @@
                DisplayName="Annotations" />
   </EnumProperty>
 
+  <BoolProperty Name="Prefer32Bit"
+                DisplayName="Prefer 32-bit"
+                Description="Set the 32-bit preferred flag for the application. Only applies to executables."
+                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2166468"
+                Category="General">
+    <BoolProperty.Metadata>
+      <NameValuePair Name="VisibilityCondition">
+        <NameValuePair.Value>(has-net-framework)</NameValuePair.Value>
```

Comment:
It seems like we should extend the visibility condition to test whether `OutputType` is not `Library` (removing the need for "Only applies to executables" in the description.

Reply:
Added to visibility condition.